### ISSUE TITLE
AspNetCore bootstrapper

### ISSUE THREAD

**Issue by Jetski5822** [khellang/nancy-bootstrapper-prototype#1]
Hey, was looking for a Nancy bootstrapper for AspNetCore, was wondering if this branch will ever make it to Nancy core as it looks interesting.

If not, do you have an alternative?

**Comment by khellang** [comment]
Yes. Maybe not in this exact form, but it will definitely make its way back pretty soon. We're in the process of migrating back and cleaning up our csproj's. After that's done, we'll start looking at bringing in the new bootstrapper that's compatible with ASP.NET Core :smile:

**Comment by Jetski5822** [comment]
Okay cool, I am trying to hook it in to OrchardCore and at the same time trying to avoid multiple ioc containers :)

**Comment by sloncho** [comment]
Hey, good stuff. Any ETA for this to go back into clinteastwood?

**Comment by khellang** [comment]
Hey! No ETA, but we want to work this into 2.0, probably for the `dangermouse` release (the one after `clinteastwood`).

**Comment by sloncho** [comment]
Looks like a lot of changes in how the engine is done. Where is the dangermouse? Can I help?

**Comment by sdmimaye** [comment]
Hallo. Just wanted to give it a try? Are you guys still working on it?

**Comment by Jetski5822** [comment]
Double bump ^^^
